spec/routines/search_users_spec.rb

```diff
@@ -124,4 +124,4 @@
 
   end
 
-end
\ No newline at end of file
+end
```

Comment:
You probably already figured out what was wrong with the .first, but it's because Postgresql sorts differently.

You could fix it by specifying a sort in the query, i.e.  `user_1.contact_infos.email_addresses.order(:value).first`  

Marking them all verified is probably the right thing do though, so that's moot.